kvm/v15.0.0/release.yaml

```
@@ -0,0 +1,55 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v15.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.8.0
+  - name: chart-operator
+    version: 2.19.0
+  - componentVersion: 1.8.3
+    name: coredns
+    version: 1.6.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.4.0
+  - componentVersion: 0.4.1
+    name: metrics-server
+    version: 1.4.0
+  - name: net-exporter
+    version: 1.10.3
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.8.0
+  components:
+  - name: app-operator
+    releaseOperatorDeploy: false
+    version: 5.2.0
+  - name: calico
+    version: 3.19.2
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 1.0.1
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.27.1
```

```suggestion
    version: 0.27.2
```

@tfussell This has the fix for the app-operator config problem.

---

Thanks for the quick turnaround. I'll retest shortly.